api/test/docker/apisix_config.yaml

```diff
@@ -57,61 +54,14 @@ apisix:
       - 10095
 
 nginx_config:
-  error_log_level: "debug"
-
-plugins:                          # plugin list (sorted in alphabetical order)
-  - api-breaker
-  - authz-keycloak
-  - basic-auth
-  - batch-requests
-  - consumer-restriction
-  - cors
-  - echo
-  - fault-injection
-  - grpc-transcode
-  - hmac-auth
-  - http-logger
-  - ip-restriction
-  - jwt-auth
-  - kafka-logger
-  - key-auth
-  - limit-conn
-  - limit-count
-  - limit-req
-  - node-status
-  - openid-connect
-  - prometheus
-  - proxy-cache
-  - proxy-mirror
-  - proxy-rewrite
-  - redirect
-  - referer-restriction
-  - request-id
-  - request-validation
-  - response-rewrite
-  - serverless-post-function
-  - serverless-pre-function
-  - skywalking
-  - sls-logger
-  - syslog
-  - tcp-logger
-  - udp-logger
-  - uri-blocker
-  - wolf-rbac
-  - zipkin
-  - server-info
+  error_log_level: debug
 
 plugin_attr:
   skywalking:
-    service_name: APISIX
-    service_instance_name: "APISIX Instance Name"
     endpoint_addr: http://172.16.238.50:12800
   server-info:
-    report_interval: 60
     report_ttl: 3600
   prometheus:
-    export_uri: /apisix/prometheus/metrics
     enable_export_server: true
     export_addr:
-      ip: "0.0.0.0"
-      port: 9091
+      ip: 0.0.0.0
```

Comment:
Instead of modifying `apisix_config.yaml` for now, we are now using `apisix_config_v3.yaml`.